docs/guides/auth.md

```diff
@@ -326,3 +343,31 @@ $opts = [
 ];
 $client = new helloworld\GreeterClient('greeter.googleapis.com', $opts);
 ```
+
+###Authenticating with Google (Python)
+
+####Base case - No encryption/authentication
+```python
+channel = implementations.insecure_channel('localhost', 50051)
+stub = helloworld_pb2.beta_create_Greeter_stub(channel)
+...
+```
+
+###Authenticate using OAuth2 token (legacy approach)
```

Comment:
@anthmgoogle: I think what happened was that @soltanmm and I were unclear on the work to be done, @tbetbetbe volunteered to take it on, and then it never got pushed back to us when he took on other work. I think @soltanmm would be willing to return to it if the questions he had when last working on it were answered.

Let's hash out the specifics of the work in an issue in https://github.com/google/oauth2client and then implement it?